plugin-api/src/main/java/com/megaease/easeagent/plugin/api/config/IConfigFactory.java

```diff
@@ -18,6 +18,13 @@
 package com.megaease.easeagent.plugin.api.config;
 
 public interface IConfigFactory {
+    /**
+     * Returns the global configuration of this Java agent.
+     *
+     * @return The global configuration of this Java agent.
+     */
+    Config getConfigs();
```

Comment:
Config getConfig() or Configs getConfigs()

Reply:
Is it better to return an Interface instead of implementation?

Reply:
I said names, not interfaces or implementations.